From 45f455df5bb54ccc888ac0006fdeb64d1b2f7ab0 Mon Sep 17 00:00:00 2001
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Fri, 26 Feb 2021 20:21:17 +0800
Subject: [PATCH] arm64/mpam: debug: remove debug pr_info at schemata
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 45f455df5bb54ccc888ac0006fdeb64d1b2f7ab0


hulk inclusion
category: feature
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/kernel/mpam_ctrlmon.c | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm64/kernel/mpam_ctrlmon.c b/arch/arm64/kernel/mpam_ctrlmon.c
index 3fa095cfce7e..e8945c4d1851 100644
--- a/arch/arm64/kernel/mpam_ctrlmon.c
+++ b/arch/arm64/kernel/mpam_ctrlmon.c
@@ -366,16 +366,6 @@ int resctrl_group_mondata_show(struct seq_file *m, void *arg)
 	}
 
 	md.priv = of->kn->priv;
-	pr_info("%s: resname %s, rid %d, domid %d, partid %d, pmg %d, (group: partid %d, pmg %d, mon %d)\n",
-		__func__, resname,
-		md.u.rid,
-		md.u.domid,
-		md.u.partid,
-		md.u.pmg,
-		rdtgrp->closid,
-		rdtgrp->mon.rmid,
-		rdtgrp->mon.mon
-	       );
 
 	r = &resctrl_resources_all[md.u.rid];
 	rr = r->res;
@@ -468,7 +458,6 @@ static int mkdir_mondata_subdir(struct kernfs_node *parent_kn,
 		return ret;
 	}
 
-
 	/* [FIXME] Could we remove the MATCH_* param ? */
 	rr->mon_write(d, prgrp, true);
 
@@ -556,7 +545,6 @@ mongroup_create_dir(struct kernfs_node *parent_kn, struct resctrl_group *prgrp,
 	struct kernfs_node *kn;
 	int ret;
 
-	pr_info("%s: create dir %s\n", __func__, name);
 	/* create the directory */
 	kn = kernfs_create_dir(parent_kn, name, parent_kn->mode, prgrp);
 	if (IS_ERR(kn)) {
-- 
2.26.2

